services/frontend-v3/src/i18n/en_CA.json

```
@@ -356,6 +357,12 @@
   "profile.do.not.specify": "Do not specify",
   "profile.not.specified": "Not specified",
   "profile.not.found": "Invalid profile URL",
+
+  "profile.geds.changes": "Confirm changes",
+  "profile.geds.sync.button": "Sync GCDirectory Data",
```

Acc. to https://gcconnex.gctools-outilsgc.ca/en/support/solutions/articles/2100027174-how-do-i-sync-my-account-with-gcdirectory-

```suggestion
  "profile.geds.sync.button": "Sync GCdirectory Data",
```